source/documentation/doc-Release_Notes/topics/ref-ovirt-4.5.0-1.adoc

```diff
@@ -196,6 +196,9 @@ Windows 11 is supported as a guest operating system.
 BZ#link:https://bugzilla.redhat.com/1999698[1999698]::
 The Apache HTTPD SSLProtocol configuration is managed by crypto-policies instead of being set by `engine-setup`.
 
+BZ#link:https://bugzilla.redhat.com/2012830[2012830]::
+You can now use the Logical Volume Management (LVM) devices file for managing storage devices instead of LVM filter, which can be complicated to set up and difficult to manage. Starting with RHEL 9, this will be the default for storage device management.
```

Comment:
@dcdacosta  need to remove mention of RHEL 9 - not supported for any of RHV 4.4

Reply:
Good catch. Thanks @emarcusRH. I'll remove the statement.